Gem/Code/Source/GameLift/GameLiftServerComponent.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of
+ * this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/std/parallel/binary_semaphore.h>
+#include <AzFramework/Session/SessionNotifications.h>
+
+namespace AzNetworking
+{
+    class INetworkInterface;
+}
+
+namespace MultiplayerSample
+{
+    class GameLiftServerSystemComponent
```

Comment:
nit: Same as above, consider MultiplayerGameLiftSampleServer... or MPSGameLiftServer...

Class needs comment wrt to description/responsibilities.